visualization/src/main/kotlin/com/enigmashowdown/visual/LevelCountdown.kt

```diff
@@ -50,6 +76,35 @@ class LevelCountdown(
             logger.info("Ticks until begin: {}", previousState.ticksUntilBegin)
```

Comment:
I'm thinking you could remove this if statement and also the `consider updating a stage here` and `If you have images as member variables...` comments.